yo1k/tic_tac_toe/ai/ai.py

```diff
@@ -0,0 +1,47 @@
+from random import Random
+from yo1k.tic_tac_toe.kernel.action_queue import DefaultActionQueue
+from yo1k.tic_tac_toe.kernel.game import (
+    Action,
+    PlayerID,
+    State,
+    Phase,
+    Cell,
+    AI)
+
+
+class RandomAI(AI):
+    def __init__(self, player_id: PlayerID, seed: int, action_queue: DefaultActionQueue):
+        self._player_id: PlayerID = player_id
+        self._rng: Random = Random(seed)
+        self._action_queue: DefaultActionQueue = action_queue
+
+    def act(self, state: State) -> None:
+        if state.phase is Phase.BEGINNING \
+                or state.phase is Phase.OUTROUND:
+            self.act_beginning_outround(state)
+        elif state.phase is Phase.INROUND:
+            self.act_inround(state)
+        else:
+            assert False
+
+    def act_beginning_outround(self, state: State) -> None:
+        if self._player_id in state.required_ready:
+            self._action_queue.add(Action.new_ready())
+
+    def act_inround(self, state: State) -> None:
```

Comment:
These methods are not supposed to be public.

Reply:
Done.